PySDM/attributes/impl/intensive_attribute.py

```diff
@@ -12,5 +12,4 @@ def __init__(self, builder, name: str, base: str):
         super().__init__(builder, name, dependencies=(self.volume, self.base))
 
     def recalculate(self):
-        self.data.idx = self.volume.data.idx
```

Comment:
why is this not needed anymore?

Reply:
IIUC, all attributes should share one `idx` instance: https://github.com/open-atmos/PySDM/blob/587163ae7567a3323df0106315e87f42c5d8459d/PySDM/impl/particle_attributes_factory.py#L21

(and now this is done only in these four files, while we have much more attributes)

Reply:
(I meant that what is being removed in this PR is done now in just a few out of many attributes)